README.md

```diff
@@ -1 +1,95 @@
-# layout
\ No newline at end of file
+# Wolox Layout
+
+[![circle-ci](https://img.shields.io/circleci/project/github/Wolox/wolox-layout.svg)](https://circleci.com/gh/Wolox/wolox-layout)
+[![npm](https://img.shields.io/npm/v/wolox-layout.svg)](https://www.npmjs.com/package/wolox-layout)
+
+[![FEArmy](./assets/FEA_icon.png)](https://github.com/orgs/Wolox/teams/front-end-army/members)
+
+## Features
+
+* `margins.scss` & `paddings.scss` files contains all the margin and padding classes you need to start a basic project.
+* `layout.scss` contains the basic flex box properties.
+
+## Installation
+
+1.  intall the package with `npm` or `yarn`
+
+#### npm
+
+```bash
+npm i wolox-layout
+```
+
+#### yarn
+
+```bash
+yarn add wolox-layout
+```
+
+2- Include the wolox-bootstrap.scss before the index file of your CSS/SCSS.
```

Comment:
wolox-bootstrap? why don't we have `margins` and `layout` as 2 separate imports?

Reply:
I agree, we should have them separated for the better API :smiley: